### DISCUSSION TITLE
[Invalid report] Missing change in docs

### DISCUSSION THREAD

**Discussion by laoergege** [vuepress/core#743] [Q&A] [answered]
刚从 beta.26 更到 beta.36，发现有些 API 变动没说明，比如 templateSSR => templateBuild 等。折腾了我好一会儿，希望大佬们，能及时更新文档。

**Comment by Mister-Hope** [comment, marked as answer]
Your report is probably invalid

![image](https://user-images.githubusercontent.com/33315834/157072944-9ebbda1e-9eb6-4ef8-bfc2-aac8119336ec.png)

![image](https://user-images.githubusercontent.com/33315834/157072106-4f471188-ccba-42ec-bdcd-20eaa1ad0b86.png)

![image](https://user-images.githubusercontent.com/33315834/157072142-18453b8b-e783-4466-a63d-a10cae2db762.png)

I just want to confirm how it happends, so could you please check if you have a service worker registed? 

Follow thiese Steps:

1. F12
2. Click the Appication tab (or  应用 in Chinese)
3. Click "service workers", and see if there is a service worker.

**Reply by laoergege** [reply]
Oh, very thank you, you are such an active big cutie.

**Comment by Mister-Hope** [comment]
@meteorlxy This is probably the thing I reported to you in wechat. It will confuse a lot of developers coming from early V2.
